static.go

```diff
@@ -53,13 +54,68 @@ var LOGINPAGE = `{{define "login"}}
     <br>
     <div id="links">
       <section>
-        <a id="forgot-password" href={{.OSFForgotPassword}}>Forgot Your Password?</a><br>
-        <a id="institution-login" href={{.OSFInstitutionLogin}}>Login Through Your Institution</a><br>
         <a id="back-to-osf" href={{.OSFDomain}}>Back to OSF</a><br>
-        <a id="create-account" href={{.OSFCreateAccount}}>Create Account</a>
+        <a id="create-account" href={{.CASRegister}}>Create Account</a>
       </section>
     </div>
   </body>
 </html>
 
-{{end}}`
+{{end}}
+
+{{define "register"}}
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Open Science Framework | Sign Up</title>
+  </head>
+
+  <body>
+    <div id="header">
+      <span><h3>Open Science Framework | fakeCAS</h3></span>
+      <br>
+    </div>
+
+    {{if .NotAuthorized}}
```

Comment:
Any reason to not make this its own template?

Reply:
No ... I will make a template for this. Both `login` and `register` will land on this one when service is unauthorized.